apps/web/src/components/ui/download-progress.tsx

```
@@ -0,0 +1,84 @@
+				) : (
+					<Loader2 className="text-primary h-5 w-5 animate-spin" />
+				)}
+
+				<div className="min-w-0 flex-1">
+					<div className="flex items-center justify-between">
+						<p className="truncate text-sm font-medium">{error ? "Download Failed" : fileName || "Downloading..."}</p>
+						{isDownloading && onCancel && (
+							<button
+								onClick={onCancel}
+								className="text-muted-foreground hover:text-foreground text-xs transition-colors"
+							>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)